messages.rb

```diff
@@ -547,6 +646,10 @@ def respond(event)
   send_level_name(event) if msg =~ /\blevel name\b/i
   send_level_id(event) if msg =~ /\blevel id\b/i
   send_diff(event) if msg =~ /diff/i
+  send_community(event) if msg =~ /community/i && msg !~ /retirement/i
```

Comment:
`"retirement"`?

Reply:
Ahh. That makes sense.

One possibility is maybe to trim level names and usernames (so things after the word 'for', mostly) before doing this match. I think those are likely to be the only things that cause problems, though  I may be wrong about that... either way, it would avoid all of those problems, which are always embarrassing when they come up ;)

Either way, if we do leave it like this, 'I expected that to be enigmatic' means it's a prime spot for a nice comment in my opinion.